backend/routes/profile.py

```
@@ -0,0 +1,93 @@
+from api.v1.endpoints.auth import get_current_user  # Authentication dependency
+import os
+import uuid
+from pathlib import Path
+
+router = APIRouter()
+
+# ✅ Predefined fields for validation
+RELEVANT_FIELDS = [
+    "Computer Science", "Electrical Engineering", "Mechanical Engineering",
+    "Civil Engineering", "Artificial Intelligence", "Data Science", "Robotics",
+    "Physics", "Mathematics", "Economics", "Biotechnology"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)